test/tests/combobox_autocomplete-none.js

```diff
@@ -178,6 +200,9 @@ ariaTest('Test down key press with focus on textbox',
     // Check that the active descendent focus is correct
     await assertAriaSelectedAndActivedescendant(t, ex.textboxSelector, ex.optionsSelector, 0);
 
+   // Check if aria-selected is set on first item
+    await assertAttributeValues(t, ex.optionsSelector + ':nth-of-type(1)', 'aria-selected', 'true');
```

Comment:
@jongund -- aria-selected is actually checked in assertAriaSelectedAndActivedescendant, the line above! So you don't need to add this line.